app/services/nomis/booker.rb

```diff
@@ -0,0 +1,56 @@
+module Nomis
+  class Booker
+    ALREADY_PROCESSED = 'Duplicate post'.freeze
+
+    def initialize(visit)
+      self.visit = visit
```

Comment:
I'm this class doesn't really need to know about the the ActiveRecord Visit model, it would be much nice to pass in the `booking_params` and have the caller know about the know about the visit rather then have both know about the internals of the `Visit` ActiveRecord object